src/types/interfaces.ts

```diff
@@ -3,6 +3,11 @@ export enum UserType {
   CLIENT_USER = 'cu',
 }
 
+export enum View {
+  LIST_VIEW,
+  BOARD_VIEW,
+}
+
```

Comment:
Does this task depend on the ViewSettings API? If so you can use the View enum from `@prisma/client` defined here. This way we don't implement this separately from the API and its generally more typesafe.

![image](https://github.com/copilot-platforms/tasks-app/assets/59971845/407261aa-64d8-4ff1-973d-71c560f04ea2)

Reply:
Yeah, I am thinking about using the same type as of the API. However, since the API is not merged yet, I am unaware about the types being used. I will implement this during the API integration.